public/app/features/dashboard-scene/scene/DashboardScene.tsx

```diff
@@ -804,6 +810,13 @@ export class DashboardScene extends SceneObjectBase<DashboardSceneState> impleme
     return Boolean(this.getManagerKind() === ManagerKind.Repo);
   }
 
+  managerAllowsEdits() {
+    return (
+      this.isManaged() &&
+      (!this.isManagedRepository() || !this.state.meta.k8s?.annotations?.[AnnoKeyManagerAllowsEdits])
```

Comment:
I think I got a bit confused here when suggesting this change, I think it'd be 
```tsx
this.isManaged() &&
      (!this.isManagedRepository() && !this.state.meta.k8s?.annotations?.[AnnoKeyManagerAllowsEdits])
```
We need to check for managed dashboards which do not have a `repo` manager kind and do not have `AnnoKeyManagerAllowsEdits`, as either of those allows editing provisioned dashbaords from the UI. Sorry for the confusion!